sdk/storage/azure-storage-file-share/azure/storage/fileshare/_directory_client.py

```diff
@@ -39,7 +42,7 @@
 
 if TYPE_CHECKING:
     from azure.core.credentials import AzureNamedKeyCredential, AzureSasCredential, TokenCredential
-    from ._models import DirectoryProperties, Handle, NTFSAttributes
+    from ._models import FileProperties, DirectoryProperties, Handle, NTFSAttributes
```

Comment:
nit: I assume that you will go back through to the imports in a future PR but just nit that `DirectoryProperties` comes before `FileProperties` and then there are some imports above that need to be alphabetized (probably in a later PR but just making you aware 😄)